app/test_split_integer.py

```diff
@@ -1,21 +1,46 @@
+import pytest
+
 from app.split_integer import split_integer
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (
+        sum(split_integer(100, 3)) == 100
+    ), "Sum of parts is not equal to the original value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
-
-
-def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert (
+        split_integer(100, 4).count(25) == 4
+    ), "Value not split into equal parts"
+
+
+@pytest.mark.parametrize(
+    "value,number_of_parts,expected_result",
+    [
+        (10, 1, [10]),
+        (10, 10, [1] * 10)
```

Comment:
this doesn't split into one part - it actually splits in 10 parts. please remove